### ISSUE TITLE
Hello bank does not work anymore

### ISSUE THREAD

**Issue by ooii** [bnjbvr/kresus#298]
Hi,

Updated to the final version in cozy and lost the ability to add Hello Bank account.
Here is the log:

```
[2015-12-09 15:33:06:378] warn - sources/weboob | Weboob error, stderr: No handlers could be found for logger "backend.hellobank.browser"
Debug data will be saved in this directory: /tmp/weboob_session_rhvPjl
Unknown error of type <class 'weboob.exceptions.BrowserHTTPNotFound'>

[2015-12-09 15:33:06:379] error - controllers/accesses | The access process creation failed, cleaning up...
[2015-12-09 15:33:06:434] warn - http-error | no status in asyncErr
Error
    at asyncErr (/usr/local/cozy/apps/kresus/build/server/helpers.js:34:46)
    at create$ (/usr/local/cozy/apps/kresus/build/server/controllers/accesses.js:230:75)
    at tryCatch (/usr/local/cozy/apps/kresus/node_modules/babel-runtime/regenerator/runtime.js:72:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/usr/local/cozy/apps/kresus/node_modules/babel-runtime/regenerator/runtime.js:334:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/usr/local/cozy/apps/kresus/node_modules/babel-runtime/regenerator/runtime.js:105:21)
    at tryCatch (/usr/local/cozy/apps/kresus/node_modules/babel-runtime/regenerator/runtime.js:72:40)
    at invoke (/usr/local/cozy/apps/kresus/node_modules/babel-runtime/regenerator/runtime.js:146:20)
    at /usr/local/cozy/apps/kresus/node_modules/babel-runtime/regenerator/runtime.js:154:13
    at run (/usr/local/cozy/apps/kresus/node_modules/core-js/library/modules/es6.promise.js:104:47)
    at /usr/local/cozy/apps/kresus/node_modules/core-js/library/modules/es6.promise.js:115:28
[2015-12-09 15:33:06:434] error - http-error | Error: when creating a bank access: [object Object] - when creating a bank access: HTTP Error 404: Not Found (url="/NS_VIRDF?Origine=DSP_VIR&stp=20151209153306")
[2015-12-09 15:33:24:861] warn - sources/weboob | Weboob error, stderr: No handlers could be found for logger "backend.hellobank.browser"
Debug data will be saved in this directory: /tmp/weboob_session_stEwGB
Unknown error of type <class 'weboob.exceptions.BrowserHTTPNotFound'>

```

**Comment by bnjbvr** [comment]
Thanks for the report! It actually looks like an issue with weboob. Can you try updating it in the settings section?

**Comment by ooii** [comment]
Updated and even reinstalled with same result.

**Comment by ZeHiro** [comment]
It is a weboob problem, there is a fix but it is not yet backported in the official repos (seen this on #weboob irc).

**Comment by bnjbvr** [comment]
Hi @ooii, can you try updating Weboob and see if it closes the issue?

**Comment by ooii** [comment]
Updating weboob from Kresus settings and even reinstalling it did not fix it.
Note that I'm using weboob in Cozy.

**Comment by bnjbvr** [comment]
@ooii, can you retry again, please? I've got reports that HelloBank has been working again, since today or so.

**Comment by ooii** [comment]
I'm leaving for vacation in few minutes. I'll test on the 2nd of january. Thanks for all the help. I wish you the best for 2016.

**Comment by ooii** [comment]
Hi,
Still not working. Updated Kresus, weboob and reinstalled it. Tried to use BNP Paribas but does not accept my password.

**Comment by bnjbvr** [comment]
Ha, great that you're around. I actually have a fix on a branch: can you follow the indications in https://forum.cozy.io/t/kresus-hello-bank-non-supportee/1787/6?u=ben and let me know if you hit an error?

**Comment by ooii** [comment]
Yes, it works.

**Comment by bnjbvr** [comment]
Great to hear, thank you for testing! Yes, we will probably release a new version of Kresus in the next few days, so it'd be better to keep this issue open in the meanwhile.

**Comment by bnjbvr** [comment]
Actually closing, as there's a fix and it references #313.
